spec/unit/errr-from-error-spec.js

```diff
@@ -1040,11 +1036,74 @@ describe("Given the errr module", function () {
             expect(err).eql(undefined);
 
             let stack = `Error: [${context.uniqueId}] Some Error`,
-              stringifiedDebugParams = `${constants.DebugPrefix}${JSON.stringify(context.debugParams, Private.replacer, 2)}`;
+              stringifiedDebugParams = `${constants.DebugPrefix}${JSON.stringify(context.debugParams, JsonReplacer.replace, 2)}`;
```

Comment:
these diffs are awful. only changes made:
- this line
- the new pretty-print test below.